frontend/src/Pages/ResourcesManagement.tsx

```diff
@@ -147,21 +214,55 @@ export default function ResourcesManagement() {
     };
 
     const updateFinalState = async (e: React.MouseEvent) => {
+        setToast({ state: ToastState.null, message: '' });
         e.preventDefault();
         const newCollectionList = collectionList.map((c, i) => {
             c.rank = i + 1;
             c.id = i;
             return c;
         });
-        const response = await API.put('left-menu', newCollectionList);
-        if (response.success) {
-            await mutate();
-            toaster('Collections Saved!', ToastState.success);
-        } else {
-            toaster(
-                'All collections must have associated links',
-                ToastState.error
+        try {
```

Comment:
Can we try and get this to work with `API.put` instead of `axios.put`? We recently refactored a lot of the components and pages to rely on the` API` library to minimize the amount of code spent error checking and validating response status codes.

Reply:
@MillerSenior Is there a reason you had to use `axios` instead of `API`? We're trying to make most of our API calls through the `API` library so that we don't have repetitive error-handling code.